lib/engine/game/g_1880/step/buy_sell_par_shares.rb

```diff
@@ -128,7 +128,7 @@ def process_par(action)
 
           def process_buy_shares(action)
             super
-            @game.receive_capital(action.corporation) if @game.full_cap_event
+            @game.receive_capital(action.bundle.corporation) if @game.full_cap_event
```

Comment:
There are two entries into receive_capital:
1- when event triggers on train 3 in an OR
2- future SRs after train 3

to gain capital, a crop needs to have its fifth share sold AND not have gotten the half cap previously (to avoid double dipping)

this gain capital check is done in the method itself, so that i don't have to do that check in the event trigger! method
I could add @game.full_cap_event to the method itself, and call it without a condition in buy_sell_par_shares, but this seems more readable